erpnext/stock/doctype/material_request/material_request_list.js

```diff
@@ -1,24 +1,25 @@
 frappe.listview_settings['Material Request'] = {
 	add_fields: ["material_request_type", "status", "per_ordered", "per_received", "transfer_status"],
 	get_indicator: function(doc) {
-		if(doc.status=="Stopped") {
+		var precision = frappe.defaults.get_default("float_precision");
+		if (doc.status=="Stopped") {
 			return [__("Stopped"), "red", "status,=,Stopped"];
-		} else if(doc.transfer_status && doc.docstatus != 2) {
+		} else if (doc.transfer_status && doc.docstatus != 2) {
 			if (doc.transfer_status == "Not Started") {
 				return [__("Not Started"), "orange"];
 			} else if (doc.transfer_status == "In Transit") {
 				return [__("In Transit"), "yellow"];
 			} else if (doc.transfer_status == "Completed") {
 				return [__("Completed"), "green"];
 			}
-		} else if(doc.docstatus==1 && flt(doc.per_ordered, 2) == 0) {
+		} else if (doc.docstatus==1 && flt(doc.per_ordered, precision) == 0) {
```

Comment:
problems here: 
1) if my precision is set to 2 in System Settings, we will still face this problem. `flt(99.9975, 2) = 100`
2) Let assume default precision is 3 , this works, status is Partially Ordered. But the per_ordered field still shows **100%**. This one is a frappe problem in Percent field formatter. 
![Screenshot 2020-12-10 at 9 37 09 PM](https://user-images.githubusercontent.com/25857446/101796823-59248780-3b2f-11eb-9d4d-e9323b7c3c0c.png)

This fix seems okay, but maybe fixing it on frappe too would perhaps complete the fix. Right now it is incomplete. You can check out the `Percent` formatter in `formatters.js` in Frappe. It rounds to two places only, maybe we can use precision there.

Reply:
@marination I have already fixed the Percent field precision in frappe, https://github.com/frappe/frappe/pull/12010